### DISCUSSION TITLE
Can approach circles not be affected by screen scaling?

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#30628] [Q&A] [answered]
Is there any interest from other players to make it so that the approach circles don't get cut off when using screen scaling? it looks so weird to have these be cut off when there is room left on the screen where it could be displayed. 
![image](https://github.com/user-attachments/assets/b7e1d39d-8f88-4aa1-9c30-909c760af44a)

**Comment by Joehuu** [comment, marked as answer]
This is already tracked as an issue in https://github.com/ppy/osu/issues/5353 (all objects can get cut off and is still undecided if it's intentional or not).
